File: surf-api-bukkit/surf-api-bukkit-api/src/main/java/dev/slne/surf/surfapi/bukkit/api/nms/listener/packets/serverbound/NmsServerboundPacket.java
package dev.slne.surf.surfapi.bukkit.api.nms.listener.packets.serverbound;

import dev.slne.surf.surfapi.bukkit.api.nms.listener.packets.NmsPacket;
import org.jetbrains.annotations.ApiStatus.NonExtendable;

@NonExtendable
public sealed interface NmsServerboundPacket extends NmsPacket permits RenameItemPacket,
    SignUpdatePacket {

}

File: surf-api-bukkit/surf-api-bukkit-api/src/main/kotlin/dev/slne/surf/surfapi/bukkit/api/nms/listener/packets/serverbound/NmsServerboundPacket.kt
package dev.slne.surf.surfapi.bukkit.api.nms.listener.packets.serverbound

import dev.slne.surf.surfapi.bukkit.api.nms.NmsUseWithCaution
import dev.slne.surf.surfapi.bukkit.api.nms.listener.packets.NmsPacket

@NmsUseWithCaution
sealed interface NmsServerboundPacket : NmsPacket
